packages/agoric-cli/src/commands/wallet.js

```diff
@@ -58,12 +58,12 @@ export const makeWalletCommand = async () => {
           ...networkConfig,
         });
       } else {
-        const params = fetchSwingsetParams(networkConfig);
+        const params = await fetchSwingsetParams(networkConfig);
         assert(
-          params.power_flag_fees.length === 1,
-          'multiple power_flag_fees not supported',
+          params.powerFlagFees.length === 1,
+          'multiple powerFlagFees not supported',
         );
-        const { fee: fees } = params.power_flag_fees[0];
+        const { fee: fees } = params.powerFlagFees[0];
```

Comment:
Was this an expected snake_case to camelCase change, or was the old code broken?

Reply:
Broken with cosmic-proto 0.4.0. I don't have reason to think it was broken in 0.3.0 but I don't know whether it was tested.